autonomy/analyse/abci/app_spec.py

```diff
@@ -18,18 +18,23 @@
 # ------------------------------------------------------------------------------
 """Generates the specification for a given ABCI app in YAML/JSON/Mermaid format."""
 
+
 import importlib
+import inspect
 import json
 import logging
 import re
 import sys
+import textwrap
 from collections import defaultdict, deque
 from itertools import product
 from pathlib import Path
 from typing import Any, Dict, List, OrderedDict, Set, TextIO, Tuple
 
 import yaml
 
+from packages.valory.skills.abstract_round_abci.base import AbciApp
```

Comment:
This is an absolute no-go ...this doesn't necessarily exist from perspective of the autonomy

Reply:
It's (somewhat hackily) solved in `check_one` method